boot/freeldr/freeldr/arch/uefi/uefihw.c

```diff
@@ -123,7 +123,8 @@ DetectAcpiBios(PCONFIGURATION_COMPONENT_DATA SystemKey, ULONG *BusNumber)
 }
 
 PCONFIGURATION_COMPONENT_DATA
-UefiHwDetect(VOID)
+UefiHwDetect(
+    _In_opt_ PCSTR Options)
```

Comment:
Hmm should i be doing something here?

Reply:
I think this function should at least detect the PS/2 and display controllers. The `Options` parameter can be safely ignored in an UEFI environment, and that doesn't cause any problems.